examples/examples_multidim/003_multi_dimensional_cp_ct_TI.py

```diff
@@ -0,0 +1,45 @@
+"""Example: Multi-dimensional power/thrust coefficients with turbulence intensity
+This example follows the previous example, but demonstrating how a multidimensional turbine can be
+used to model the effect of turbulence intensity on power and thrust coefficient.
+
+NOTE: The multi-dimensional power/thrust coefficient data used in this example is fictional for the
+purposes of facilitating this example.
```

Comment:
Turbine 0's power is increased by turbulence, even near the knee of the power curve.  You point out a few times this is fake data but could also say "... and that is why the changes in power curve due to TI don't follow usual behaviors..."